pkg/compact/compact.go

```diff
@@ -501,10 +501,6 @@ func (cg *Group) compact(ctx context.Context, dir string, comp tsdb.Compactor) (
 			}
 			uniqueSources[s] = struct{}{}
 		}
-		// Ensure all input blocks are valid.
-		if err := block.VerifyIndex(filepath.Join(pdir, "index")); err != nil {
```

Comment:
mhm I think we need more testing for that, can you drop it from this PR to merge just bucket cmd changes (they are ok)

Reply:
Done!